eclair-core/src/main/scala/fr/acinq/eclair/channel/fsm/ChannelOpenDualFunded.scala

```diff
@@ -584,10 +579,8 @@ trait ChannelOpenDualFunded extends DualFundingHandlers with ErrorHandlers {
               Seq(channelReady)
           }
           goto(WAIT_FOR_DUAL_FUNDING_READY) using DATA_WAIT_FOR_DUAL_FUNDING_READY(commitments, shortIds, channelReady) storing() sending toSend
-        case None =>
-          log.error(s"internal error: the funding tx that confirmed doesn't match any of our funding txs: ${confirmedTx.bin}")
-          rollbackDualFundingTxs(allFundingTxs.map(_.fundingTx))
-          goto(CLOSED)
+        case Failure(t) =>
+          handleLocalError(t, d, Some(w))
```

Comment:
If you do that the eclair process will stop when that happens, that's not what we'd like...
I don't think we want to spend our local commitment either, that doesn't make any sense.
I don't see what else we can do than going to either `CLOSED` or `ERR_INFORMATION_LEAK` in that case (which should never happen anyway).

Reply:
I agree, it doesn't really make sense.

How about simply `stay()`? It's not really a case of `ERR_INFORMATION_LEAK` I think, more like a bug, maybe it's best to just wait.

For the sake of argument, we could almost call `.get` on the `find`, right?

Reply:
> For the sake of argument, we could almost call .get on the find, right?

That's true, but I'm always very wary of that...but just adding an error log and doing `stay()` works fine for me!